src/js/editLocalStorage.js

```diff
@@ -0,0 +1,29 @@
+// import createElement from './createElement.js';
+import List from './list.js';
+
+const editDescription = (index, icon) => {
```

Comment:
- [ ] The edit function was implemented and the changed data is stored. However, after editing a task, it is left blank on your UI until the page is refreshed. Please make sure that after editing a task the new value is displayed directly on the UI
![image](https://github.com/IvonneBenitesRodriguez/todo-list-microverse/assets/45460192/239ca45e-8f3a-4963-afac-44a1972e3536)
![image](https://github.com/IvonneBenitesRodriguez/todo-list-microverse/assets/45460192/5d1caa53-14be-4852-b837-c9b5e9ad3448)

Reply:
Dear Lucas, 
Very nice to meet you, 
Thanks for the suggestions, 
I already modified it, thanks again, 
Best regards,

Reply:
It's nice to meet you as well @IvonneBenitesRodriguez . I'm glad I helped. Happy coding :man_technologist:

Reply:
- Well resolved